legacy/src/test/java/org/openscience/cdk/smiles/DeduceBondSystemToolTest.java

```diff
@@ -29,13 +33,13 @@
 import org.openscience.cdk.interfaces.IAtom;
 import org.openscience.cdk.interfaces.IAtomContainer;
 import org.openscience.cdk.interfaces.IAtomType.Hybridization;
-import org.openscience.cdk.interfaces.IBond.Order;
 import org.openscience.cdk.interfaces.IBond;
+import org.openscience.cdk.interfaces.IBond.Order;
 import org.openscience.cdk.ringsearch.AllRingsFinder;
-import org.openscience.cdk.silent.SilentChemObjectBuilder;
 import org.openscience.cdk.silent.Atom;
 import org.openscience.cdk.silent.AtomContainer;
 import org.openscience.cdk.silent.Bond;
+import org.openscience.cdk.silent.SilentChemObjectBuilder;
```

Comment:
(yes, tests actually are fine... actually, I find it hard to see in which file changes are in the web gui... overlooked that; but this was not a change request in the first place...)